pycbc/pnutils.py

```diff
@@ -30,33 +30,36 @@
 import lal, lalsimulation
 import numpy
 from scipy.optimize import bisect, brentq, minimize
+from pycbc import derived_parameters
```

Comment:
I assume use of derived_parameters within pnutils is to just keep things temporarily working. I'm not super opposed to this, but it's obviously inelegant. If we go this route, please add an issue and someone needs to work to remove this in the near future. The functions should be all removed from pnutils.